examples/export_runs.py

```diff
@@ -0,0 +1,50 @@
+from domino import Domino
+import pandas as pd
+import json
+import os
+
+output_dir = "results"
+
+# connect to domino; be sure to have DOMINO_USER_API_KEY env set. Domino automatically includes this for you
```

Comment:
May be worthwhile to clarify that "if your code is running inside of Domino it's automatically included" - or something similar.